### ISSUE TITLE
docker container version 7.2 SSL Problems

### ISSUE THREAD

**Issue by Reis-A** [ONLYOFFICE/DocumentServer#1915]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
the server is reachable by insecure http
https is not working even though the configuration is exactly the same as in docker container 7.1.1 with the same certificate
openssl s_client -connect localhost:443 returns the error 0A000126:SSL routines::unexpected eof while reading 
I assume it has to do with the switch to openssl 3 in ubuntu 22.04

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
sudo docker run --name=onlyoffice --detach --restart=always --add-host=mynextclouddomain.de:internal ip \
        -v /app/onlyoffice/DocumentServer/logs:/var/log/onlyoffice \
        -v /app/onlyoffice/DocumentServer/data:/var/www/onlyoffice/Data \
        -v /app/onlyoffice/DocumentServer/lib:/var/lib/onlyoffice \
        -v /app/onlyoffice/DocumentServer/db:/var/lib/postgresql \
        -p 80:80 -p 443:443 \
        -i -t -d \
        -e JWT_ENABLED=true -e JWT_SECRET=********
        -e USE_UNAUTHORIZED_STORAGE=true  onlyoffice/documentserver-ee:7.2

with onlyoffice.key and onlyoffice.crt in the data folder. which is left unchanged by the switch from version 7.1.1 to 7.2
**What is the expected behavior?**
the server is reachable by https

**Did this work in previous versions of DocumentServer?**
no problems in 7.1.1. i assume it has to do with the switch tu ubuntu 2204 within the container
**DocumentServer version:**
7.2
**Operating System:**
docker 20.12 on ubuntu server LTS 22.04
**Browser version:**
NA

**Comment by neotaburiss** [comment]
**I can confirm the same problem even with ONLYOFFICE_HTTPS_HSTS_ENABLED turn to FALSE which means http are not OK either.

But with all same config/environment, version 7.1.1 works OK.**

**Docker container log as below:**

_ErrorWarningSystemArrayLogin


chmod: cannot access '/usr/share/ca-certificates/ds/*.pem': No such file or directory
/var/www/onlyoffice/documentserver/npm/json[27]: ../src/node_platform.cc:61:std::unique_ptr<long unsigned int> node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start(): Assertion `(0) == (uv_thread_create(t.get(), start_thread, this))' failed.

1: 0x8e08f0 node::Abort() [/var/www/onlyoffice/documentserver/npm/json]
2: 0x8e096c [/var/www/onlyoffice/documentserver/npm/json]
3: 0x94c4f7 node::WorkerThreadsTaskRunner::WorkerThreadsTaskRunner(int) [/var/www/onlyoffice/documentserver/npm/json]
4: 0x94c607 node::NodePlatform::NodePlatform(int, v8::TracingController*) [/var/www/onlyoffice/documentserver/npm/json]
5: 0x8bee61 node::InitializeOncePerProcess(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
6: 0x8bf19c node::Start(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
7: 0x14f04d96cd90 [/lib/x86_64-linux-gnu/libc.so.6]
8: 0x14f04d96ce40 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
9: 0x842a71 [/var/www/onlyoffice/documentserver/npm/json]
/var/www/onlyoffice/documentserver/npm/json[28]: ../src/node_platform.cc:61:std::unique_ptr<long unsigned int> node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start(): Assertion `(0) == (uv_thread_create(t.get(), start_thread, this))' failed.

1: 0x8e08f0 node::Abort() [/var/www/onlyoffice/documentserver/npm/json]
2: 0x8e096c [/var/www/onlyoffice/documentserver/npm/json]
3: 0x94c4f7 node::WorkerThreadsTaskRunner::WorkerThreadsTaskRunner(int) [/var/www/onlyoffice/documentserver/npm/json]
4: 0x94c607 node::NodePlatform::NodePlatform(int, v8::TracingController*) [/var/www/onlyoffice/documentserver/npm/json]
5: 0x8bee61 node::InitializeOncePerProcess(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
6: 0x8bf19c node::Start(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
7: 0x14b070f7cd90 [/lib/x86_64-linux-gnu/libc.so.6]
8: 0x14b070f7ce40 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
9: 0x842a71 [/var/www/onlyoffice/documentserver/npm/json]
/var/www/onlyoffice/documentserver/npm/json[29]: ../src/node_platform.cc:61:std::unique_ptr<long unsigned int> node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start(): Assertion `(0) == (uv_thread_create(t.get(), start_thread, this))' failed.

1: 0x8e08f0 node::Abort() [/var/www/onlyoffice/documentserver/npm/json]
2: 0x8e096c [/var/www/onlyoffice/documentserver/npm/json]
3: 0x94c4f7 node::WorkerThreadsTaskRunner::WorkerThreadsTaskRunner(int) [/var/www/onlyoffice/documentserver/npm/json]
4: 0x94c607 node::NodePlatform::NodePlatform(int, v8::TracingController*) [/var/www/onlyoffice/documentserver/npm/json]
5: 0x8bee61 node::InitializeOncePerProcess(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
6: 0x8bf19c node::Start(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
7: 0x14c804204d90 [/lib/x86_64-linux-gnu/libc.so.6]
8: 0x14c804204e40 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
9: 0x842a71 [/var/www/onlyoffice/documentserver/npm/json]
/var/www/onlyoffice/documentserver/npm/json[30]: ../src/node_platform.cc:61:std::unique_ptr<long unsigned int> node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start(): Assertion `(0) == (uv_thread_create(t.get(), start_thread, this))' failed.

1: 0x8e08f0 node::Abort() [/var/www/onlyoffice/documentserver/npm/json]
2: 0x8e096c [/var/www/onlyoffice/documentserver/npm/json]
3: 0x94c4f7 node::WorkerThreadsTaskRunner::WorkerThreadsTaskRunner(int) [/var/www/onlyoffice/documentserver/npm/json]
4: 0x94c607 node::NodePlatform::NodePlatform(int, v8::TracingController*) [/var/www/onlyoffice/documentserver/npm/json]
5: 0x8bee61 node::InitializeOncePerProcess(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
6: 0x8bf19c node::Start(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
7: 0x153400b58d90 [/lib/x86_64-linux-gnu/libc.so.6]
8: 0x153400b58e40 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
9: 0x842a71 [/var/www/onlyoffice/documentserver/npm/json]
/var/www/onlyoffice/documentserver/npm/json[31]: ../src/node_platform.cc:61:std::unique_ptr<long unsigned int> node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start(): Assertion `(0) == (uv_thread_create(t.get(), start_thread, this))' failed.

1: 0x8e08f0 node::Abort() [/var/www/onlyoffice/documentserver/npm/json]
2: 0x8e096c [/var/www/onlyoffice/documentserver/npm/json]
3: 0x94c4f7 node::WorkerThreadsTaskRunner::WorkerThreadsTaskRunner(int) [/var/www/onlyoffice/documentserver/npm/json]
4: 0x94c607 node::NodePlatform::NodePlatform(int, v8::TracingController*) [/var/www/onlyoffice/documentserver/npm/json]
5: 0x8bee61 node::InitializeOncePerProcess(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
6: 0x8bf19c node::Start(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
7: 0x14f216213d90 [/lib/x86_64-linux-gnu/libc.so.6]
8: 0x14f216213e40 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
9: 0x842a71 [/var/www/onlyoffice/documentserver/npm/json]
/var/www/onlyoffice/documentserver/npm/json[32]: ../src/node_platform.cc:61:std::unique_ptr<long unsigned int> node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start(): Assertion `(0) == (uv_thread_create(t.get(), start_thread, this))' failed.

1: 0x8e08f0 node::Abort() [/var/www/onlyoffice/documentserver/npm/json]
2: 0x8e096c [/var/www/onlyoffice/documentserver/npm/json]
3: 0x94c4f7 node::WorkerThreadsTaskRunner::WorkerThreadsTaskRunner(int) [/var/www/onlyoffice/documentserver/npm/json]
4: 0x94c607 node::NodePlatform::NodePlatform(int, v8::TracingController*) [/var/www/onlyoffice/documentserver/npm/json]
5: 0x8bee61 node::InitializeOncePerProcess(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
6: 0x8bf19c node::Start(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
7: 0x14f23eeb9d90 [/lib/x86_64-linux-gnu/libc.so.6]
8: 0x14f23eeb9e40 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
9: 0x842a71 [/var/www/onlyoffice/documentserver/npm/json]
/var/www/onlyoffice/documentserver/npm/json[33]: ../src/node_platform.cc:61:std::unique_ptr<long unsigned int> node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start(): Assertion `(0) == (uv_thread_create(t.get(), start_thread, this))' failed.

1: 0x8e08f0 node::Abort() [/var/www/onlyoffice/documentserver/npm/json]
2: 0x8e096c [/var/www/onlyoffice/documentserver/npm/json]
3: 0x94c4f7 node::WorkerThreadsTaskRunner::WorkerThreadsTaskRunner(int) [/var/www/onlyoffice/documentserver/npm/json]
4: 0x94c607 node::NodePlatform::NodePlatform(int, v8::TracingController*) [/var/www/onlyoffice/documentserver/npm/json]
5: 0x8bee61 node::InitializeOncePerProcess(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
6: 0x8bf19c node::Start(int, char**) [/var/www/onlyoffice/documentserver/npm/json]
7: 0x15045d600d90 [/lib/x86_64-linux-gnu/libc.so.6]
8: 0x15045d600e40 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
9: 0x842a71 [/var/www/onlyoffice/documentserver/npm/json]
/app/ds/run-document-server.sh: line 215: -3: substring expression < 0
nc: port number invalid:
Waiting for connection to the host on port
nc: port number invalid:
Waiting for connection to the host on port
nc: port number invalid:
Waiting for connection to the host on port
nc: port number invalid:
Waiting for connection to the host on port
nc: port number invalid:
Waiting for connection to the host on port
nc: port number invalid:
Waiting for connection to the host on port
nc: port number invalid:
Waiting for connection to the host on port
nc: port number invalid:
Waiting for connection to the host on port
nc: port number invalid:
Waiting for connection to the host on port
nc: port number invalid:
Waiting for connection to the host on port
nc: port number invalid:
Waiting for connection to the host on port
nc: port number invalid:
Waiting for connection to the host on port

ErrorWarningSystemArrayLogin


* Starting PostgreSQL 12 database server
...done.
* Starting RabbitMQ Messaging Server rabbitmq-server
...done.
psql:/var/www/onlyoffice/documentserver/server/schema/postgresql/removetbl.sql:4: NOTICE: table "doc_callbacks" does not exist, skipping
psql:/var/www/onlyoffice/documentserver/server/schema/postgresql/removetbl.sql:9: NOTICE: function merge_db(pg_catalog.varchar,int2,int8,pg_catalog.timestamp,pg_catalog.varchar,int8,int8) does not exist, skipping
psql:/var/www/onlyoffice/documentserver/server/schema/postgresql/removetbl.sql:10: NOTICE: function merge_db(pg_catalog.varchar,int2,int4,pg_catalog.timestamp,pg_catalog.varchar,int4,int4) does not exist, skipping
Starting supervisor: supervisord.
* Starting periodic command scheduler cron
...done.
* Starting nginx nginx
...done.
Generating AllFonts.js, please wait...Done
Generating presentation themes, please wait...Done
ds:docservice: stopped
ds:docservice: started
ds:converter: stopped
ds:converter: started
* Reloading nginx configuration nginx
...done.
==> /var/log/onlyoffice/documentserver/converter/err.log <==

==> /var/log/onlyoffice/documentserver/converter/out.log <==
[2022-09-26T01:20:57.315] [WARN] nodeJS - update cluster with 1 workers
[2022-09-26T01:20:57.324] [WARN] nodeJS - worker 1077 started.
[2022-09-26T01:20:57.328] [WARN] nodeJS - update cluster with 1 workers
[2022-09-26T01:21:40.992] [WARN] nodeJS - update cluster with 1 workers
[2022-09-26T01:21:40.998] [WARN] nodeJS - worker 1206 started.
[2022-09-26T01:21:41.000] [WARN] nodeJS - update cluster with 1 workers

==> /var/log/onlyoffice/documentserver/docservice/err.log <==

==> /var/log/onlyoffice/documentserver/docservice/out.log <==
[2022-09-26T01:20:57.937] [WARN] nodeJS - Express server starting...
[2022-09-26T01:20:57.938] [WARN] nodeJS - Set services.CoAuthoring.token.enable.browser, services.CoAuthoring.token.enable.request.inbox, services.CoAuthoring.token.enable.request.outbox in the Document Server config to prevent an unauthorized access to your documents and the substitution of important parameters in ONLYOFFICE Document Server requests.
[2022-09-26T01:20:57.939] [WARN] nodeJS - Failed to subscribe to plugin folder updates. When changing the list of plugins, you must restart the server. https://nodejs.org/docs/latest/api/fs.html#fs_availability

[2022-09-26T01:20:59.209] [WARN] nodeJS - Express server listening on port 8000 in production-linux mode. Version: 7.1.1. Build: 23

[2022-09-26T01:21:39.571] [WARN] nodeJS - Express server starting...
[2022-09-26T01:21:39.573] [WARN] nodeJS - Set services.CoAuthoring.token.enable.browser, services.CoAuthoring.token.enable.request.inbox, services.CoAuthoring.token.enable.request.outbox in the Document Server config to prevent an unauthorized access to your documents and the substitution of important parameters in ONLYOFFICE Document Server requests.
[2022-09-26T01:21:39.573] [WARN] nodeJS - Failed to subscribe to plugin folder updates. When changing the list of plugins, you must restart the server. https://nodejs.org/docs/latest/api/fs.html#fs_availability

[2022-09-26T01:21:39.680] [WARN] nodeJS - Express server listening on port 8000 in production-linux mode. Version: 7.1.1. Build: 23


==> /var/log/onlyoffice/documentserver/metrics/err.log <==

==> /var/log/onlyoffice/documentserver/metrics/out.log <==
26 Sep 01:20:57 - [1037] reading config file: ./config/config.js
26 Sep 01:20:57 - server is up INFO

==> /var/log/onlyoffice/documentserver/nginx.error.log <==


==> /var/log/onlyoffice/documentserver/spellchecker/err.log <==_

**Comment by ymotton** [comment]
We're having the same issue.
This seems related to `/var/www/onlyoffice/documentserver/npm/json` crashing on an assertion.

**Comment by Reis-A** [comment]
I solved the problem with my server.
In the folder /app/onlyoffice/documentserver/data/certs  I had obsolete files in the folder with ending crt and key that where not needed anymore. In my case it were self signed certificates tls.crt and tls.key
After I renamed those files to tls.key.bak and tls.crt.bak the docker container was reachable via https, as it should be.
Apparently Onlyoffice 7.2 handles these additional certificate files differently than onlyoffice 7.1.1 before.
This solution was inspired by this observation
[Issue502-docker-documentserver](https://github.com/ONLYOFFICE/Docker-DocumentServer/issues/502)
